libbeat/service/service_windows.go

```diff
@@ -54,6 +55,8 @@ loop:
 	return
 }
 
+const couldNotConnect syscall.Errno = 1063
```

Comment:
It would be good to document the Windows error name for 1063. It will help if I ever grep for that name again.